index.bs

```diff
@@ -9824,6 +9824,63 @@ Methods</h5>
 		</div>
 </dl>
 
+<h5 id="AudioWorkletProcessor-instantiation">
+The instantiation of AudioWorkletProcessor</h5>
+
+The construction of an {{AudioWorkletNode}} is followed by the construction
```

Comment:
Is this text normative or informative?  Seems like it's informative, since the normative bit is just the last step in the AudioWorkletNode constructor?  If so, it would be best if this section were explicitly marked informative.